src/core/ddsc/src/dds_sysdef_parser.c

```diff
@@ -2625,11 +2644,23 @@ static int proc_elem_open (void *varg, UNUSED_ARG (uintptr_t parentinfo), UNUSED
       else if (ddsrt_strcasecmp (name, "tsn_listener") == 0)
         CREATE_NODE_LIST (pstate, dds_sysdef_tsn_listener_configuration, ELEMENT_KIND_DEPLOYMENT_CONF_TSN_LISTENER, NO_INIT, fini_conf_tsn_listener, tsn_listener_configurations, dds_sysdef_tsn_configuration, ELEMENT_KIND_DEPLOYMENT_CONF_TSN, pstate->current);
       else if (ddsrt_strcasecmp (name, "data_frame_specification") == 0)
-        CREATE_NODE_LIST (pstate, dds_sysdef_tsn_data_frame_specification, ELEMENT_KIND_DEPLOYMENT_CONF_TSN_TALKER_DATA_FRAME_SPEC, NO_INIT, fini_conf_tsn_data_frame_specification, data_frame_specification, dds_sysdef_tsn_talker_configuration, ELEMENT_KIND_DEPLOYMENT_CONF_TSN_TALKER, pstate->current);
```

Comment:
Can you give a little bit more background why this changes from `LIST` to `SINGLE`? I do think it is really strange to configure TSN and then send the same data to multiple flows, and to restrict it (at least until we have a good reason to not restrict it) makes sense to me. I just want to be sure I understand the reason.

Reply:
According to DDS-XML spec. data_frame and others changed has one or no entries in configuration of TSN(Listener|Talker).